wapitiCore/language_sources/fr.po

```diff
@@ -1346,3 +1346,11 @@ msgstr ""
 #: ../main/wapiti.py:1074
 msgid "Error: Internal endpoint URL must be accessible from Wapiti!"
 msgstr ""
+
+#: ../language/vulnerability.py:145 ../language/vulnerability.py:163
+msgid "Secure Flag cookie description"
+msgstr "The secure flag is an option that can be set by the application server when sending a new cookie to the user within an HTTP Response. The purpose of the secure flag is to prevent cookies from being observed by unauthorized parties due to the transmission of a the cookie in clear text"
+
+#: ../language/vulnerability.py:146 ../language/vulnerability.py:164
+msgid "Secure Flag cookie solution"
+msgstr "While the creation of the cookie, make sure to set the Secure Flag to True"
```

Comment:
You should run generateTranslations.sh script to update *.mo files

Reply:
Fixed